articles/virtual-network/virtual-network-for-azure-services.md

```diff
@@ -36,8 +36,10 @@ Deploying services within a virtual network provides the following capabilities:
 - On-premises resources can access resources in a virtual network using private IP addresses over a [Site-to-Site VPN (VPN Gateway)](../vpn-gateway/vpn-gateway-about-vpngateways.md?toc=%2fazure%2fvirtual-network%2ftoc.json#s2smulti) or [ExpressRoute](../expressroute/expressroute-introduction.md?toc=%2fazure%2fvirtual-network%2ftoc.json).
 - Virtual networks can be [peered](virtual-network-peering-overview.md) to enable resources in the virtual networks to communicate with each other, using private IP addresses.
 - Service instances in a virtual network are fully managed by the Azure service, to monitor health of the instances, and provide required scale, based on load.
-- Service instances are deployed into a subnet in a virtual network. Inbound and outbound network access must be opened through [network security groups](security-overview.md#network-security-groups) for the subnet, per guidance provided by the services.
-- Optionally, services might require a [delegated subnet](virtual-network-manage-subnet.md#add-a-subnet) as an explicit identifier that a subnet can host a particular service. Subnet delegation gives explicit permissions to the service to create service-specific resources in the subnet.
+- Service instances are deployed into a subnet in a virtual network. Inbound and outbound network access for the subnet must be opened through [network security groups](security-overview.md#network-security-groups), per guidance provided by the service.
+- Certain injected services also impose restrictions on the subnet they are deployed in, limiting the application of policies , routes and/or combining VMs and service resources within the same subnet. Please check with each service on the specific restrictions as these may change over time based on updates to their deployment architectures. Some examples of such services are Azure NetApp Files, Dedicated HSM, Azure Container Instances, App Service, etc. 
```

Comment:
We should at least list the services that require a dedicated subnet, thoughts

Reply:
I re-did the updates on my private repo now and the associated pull request for the changes are here: https://github.com/MicrosoftDocs/azure-docs-pr/pull/74314

#please-close